package.json

```diff
@@ -51,7 +51,7 @@
     "react-native-dotenv": "^3.2.0",
     "react-native-gesture-handler": "~1.10.2",
     "react-native-maps": "0.28.0",
-    "react-native-paper": "^4.9.2",
+    "react-native-paper": "^4.8.1",
```

Comment:
If we're updating, why not use the latest? Looks to be 4.10.1
https://www.npmjs.com/package/react-native-paper

Reply:
the newest version is not necessarily better, this seems to be an ongoing bug as per the linked thread above. following the suggested answer to use this version here.